src/controllers/update/update.manage.trusts.tell.us.about.the.individual.controller.ts

```diff
@@ -67,9 +69,11 @@ export const post = async (req: Request, res: Response, next: NextFunction) => {
     const formData: IndividualTrusteesFormCommon = req.body as IndividualTrusteesFormCommon;
 
     if (!errorList.isEmpty()) {
+      const trustee = getTrustee(trust, trusteeId, TrusteeType.INDIVIDUAL) as IndividualTrustee;
```

Comment:
"as" is the developer asserting that the typing is as stated.

Reply:
Correct